requirements/constraints.txt

```diff
@@ -72,10 +72,5 @@ pytest-lazy-fixture>=999999999
 # avoid downgrades of openapi-spec-validator related libraries
 referencing<=0.35.1
 
-# Pin boto3<1.36.0 till the following is addressed https://github.com/boto/botocore/issues/2308
-# when removing this pin, also update the aws cli inside ci/github/helpers/install_aws_cli_v2.bash
```

Comment:
I think you also must update the version inside `ci/github/helpers/install_aws_cli_v2.bash`

Reply:
please verify that my previous comment, thanks.

Reply:
but why?

Reply:
as agreed. Unless some tests are failing because of it, the AWS cli will not be touched.